### PR TITLE
Fixed integration installation command

### PR DESCRIPTION
**Pull request by edshee**
## Describe changes
I implemented/fixed _ to achieve _.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to docs, I have updated the documentation accordingly.
- [x] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/mlops-stacks/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [x] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Other (add details above)

### CODE REVIEW

**Review by strickvl — APPROVED** [review]
Good catch!

**Review by fa9r — APPROVED** [review]
Looks good to me!

**Review by htahir1 — APPROVED** [review]
Good catch @edshee! We just need your signature on the CLA to get this merged!

**Comment by edshee** [comment]
@hamzamaiot Sorry, I missed that bit when opening the issue. All done.

File: docs/book/mlops-stacks/model-deployers/seldon.md [hunk 1]
```diff
@@ -280,4 +280,4 @@ to achieve this.
 
 Example of the [custom class](https://apidocs.zenml.io/0.13.0/api_docs/integrations/#zenml.integrations.seldon.custom_deployer.zenml_custom_model.ZenMLCustomModel)
 
-The built-in Seldon Core custom deployment step responsible for packaging, preparing and deploying to Seldon Core can be found [here](https://apidocs.zenml.io/0.13.0/api_docs/integrations/#zenml.integrations.seldon.steps.seldon_deployer.seldon_model_deployer_step)
\ No newline at end of file
+The built-in Seldon Core custom deployment step responsible for packaging, preparing and deploying to Seldon Core can be found [here](https://apidocs.zenml.io/0.13.0/api_docs/integrations/#zenml.integrations.seldon.steps.seldon_deployer.seldon_model_deployer_step)
```
Comment (htahir1) [inline docs/book/mlops-stacks/model-deployers/seldon.md:283]:
```suggestion
The built-in Seldon Core custom deployment step responsible for packaging, preparing and deploying to Seldon Core can be found [here](https://apidocs.zenml.io/latest/api_docs/integrations/#zenml.integrations.seldon.steps.seldon_deployer.seldon_model_deployer_step)
```

**Review by htahir1 — CHANGES_REQUESTED** [review]
Oh and I noticed one more potential issue @edshee . After this we can merge!

**Review by htahir1 — APPROVED** [review]
Alright LGTM, thanks for the catch!

**Pull request merged**